feconf.py

```diff
@@ -697,14 +697,27 @@ def get_empty_ratings():
 DEFAULT_TOPIC_SIMILARITY = 0.5
 SAME_TOPIC_SIMILARITY = 1.0
 
-SUPPORTED_SITE_LANGUAGES = {
-    'en': 'English',
-    'es': 'Español',
-    'id': 'Bahasa Indonesia',
-    'pt': 'Português',
-    'vi': 'Tiếng Việt',
-    'hi': 'हिन्दी',
-}
+# While adding another language, please ensure that the
+# languages are in alphabetical order
```

Comment:
Nit: add full stop at end.

Also, suggest prefixing the comment with "NOTE TO DEVELOPERS: " to make clear that it's intended to get read, as opposed to being just a descriptive comment. We do this elsewhere in the codebase -- you can use grep to find other examples.